multirange.js

```diff
@@ -59,8 +59,38 @@ self.multirange = function(input) {
 		ghost.style.setProperty("--high", input.valueHigh * 100 / max - 1 + "%");
 	}
 
-	input.addEventListener("input", update);
-	ghost.addEventListener("input", update);
+    function passClick(evt) {
+      // Are the ghost and input elements inverted? (ghost is lower range)
+      var isInverted = input.valueLow == ghost.value;
+      // Find the horizontal position that was clicked (as a percentage of the element's width)
+      var clickPoint = evt.offsetX / this.offsetWidth;
+      // Map the percentage to a value in the range (note, assumes a min value of 0)
+      var clickValue = max * clickPoint;
+
+      // Get the distance to both high and low values in the range
+      var highDiff = Math.abs(input.valueHigh - clickValue);
+      var lowDiff = Math.abs(input.valueLow - clickValue);
+
+      if (lowDiff < highDiff && !isInverted || (isInverted && lowDiff > highDiff)) {
+        // The low value is closer to the click point than the high value
+        // We should update the low value input
+        var passEvent = new MouseEvent("mousedown", {screenX: evt.screenX, clientX: evt.clientX});
```

Comment:
I wonder if passing `evt` as a param would work instead of constructing an object literal with a subset of properties. Have you tried it?